File: TAlaApi/TAlaApi/Models/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TAlaApi.Models
{
    public class Event
    {
        public int EventID { get; set; }

        public string Discreption { get; set; }

        public string Name { get; set; }

        public Charity owner { get; set; }

        public int moneyRange { get; set; }

        public int underProcess { get; set; }
    }
}

File: amazone_teckathon/lib/Models/Event.dart
import 'package:WeCare/Models/Charity.dart';

class Event {
  String name;
  Charity owner;
  String discription;
  int money;
  int inprosses;

  Event({this.discription, this.name, this.owner, this.money, this.inprosses});

  factory Event.fromJson(Map<String, dynamic> json) => _$productFromJson(json);
}

Event _$productFromJson(Map<String, dynamic> json) {
  return Event(
    discription: json['discreption'],
    name: json['name'],
    owner: json['owner'],
    money: json['moneyRange'],
    inprosses: json['underProcess'],
  );
}
